src/components/screens/dashboard/dashboard-screen.tsx

```diff
@@ -0,0 +1,13 @@
+import { type ReactElement } from "react";
+/**
+ * Dashboard screen component
+ * 
+ * @returns ReactElement
+ */
+function DashboardScreen (): ReactElement {
```

Comment:
Don't need the ReactElement return type here

Reply:
removed